Assets/Engine/Tile/TileRework/Connections/AdjacencyMap.cs

```diff
@@ -96,11 +96,6 @@ public bool SetConnection(Direction direction, AdjacencyData data)
 
             return changed;
         }
-        
-        public void SetConnections(AdjacencyData[] newConnections)
```

Comment:
Why do you remove this method ?

Reply:
Because nothing was using it and it is dangerous after changing to array instead of dictionary.

For a dictionary, I could loop values and only update the ones that are different, and thus ensuring the dictionary still has 8 elements, one for each direction.

For an array, the only thing I can do is test if the new array has 8 elements, but there is no way to ensure they are in correct order. Plus, if you only need to change 2 or 3 values, the person calling this method would have to KNOW that they must still pass an array of 8, which is dangerous and unnecessarily complex.

Reply:
makes sense thanks !